xAPI.md

```diff
@@ -444,13 +444,6 @@ Communities of practice could find the need to establish new Verbs, Activities,
 
 Therefore, it is expected that xAPI communities of practice generate profiles, lists, and repositories.
 
-###### Requirements for Communities of Practice
-
-* Anyone establishing a new vocabulary entry MUST own the IRI, or MUST have permission from the owner to use it to denote an
-xAPI Verb, Activity, Extension, etc.;  (This requirement cannot be enforced by an LRS, rather is meant to show the gravity 
-of coining new entries without control)
-* Anyone establishing a new vocabulary entry SHOULD make a human-readable description of the intended usage accessible at the IRI.
-
```

Comment:
This stuff is represented on line 2472 in the new version. All other additions come from here: https://github.com/adlnet/xAPI-Spec/blob/1.0.3/xAPI.md#metadata-requirements